contracts/ReferenceToken.sol

```diff
@@ -117,19 +129,29 @@ contract ReferenceToken is Ownable, ERC20, ValidatedToken {
         return true;
     }
 
-      function doSend(
-          address _from,
-          address _to,
-          uint256 _amount
-      ) private {
-          requireMultiple(_amount);
-          require(_to != address(0));          // forbid sending to 0x0 (=burning)
-          require(mBalances[_from] >= _amount); // ensure enough funds
-          requireOk(validate(_from, _to, _amount)); // Ensure passes validation
+    function doSend(
+        address _from,
+        address _to,
+        uint256 _amount
+    ) internal {
+        require(canTransfer(_from, _to, _amount));
+        requireMultiple(_amount);
+
+        mBalances[_from] = mBalances[_from].sub(_amount);
+        mBalances[_to] = mBalances[_to].add(_amount);
 
-          mBalances[_from] = mBalances[_from].sub(_amount);
-          mBalances[_to] = mBalances[_to].add(_amount);
-          Transfer(_from, _to, _amount);
-      }
+        emit Transfer(_from, _to, _amount);
+    }
 
+    function canTransfer(
```

Comment:
why don't we add requireMultiple to this method?

I don't think you'll ever be able to transfer a fraction of a token :)